app/helpers/version_settings_helper.rb

```diff
@@ -39,18 +39,23 @@ module VersionSettingsHelper
   def version_settings_fields(version, project)
     setting = version_setting_for_project(version, project)
 
-    ret = "<p>"
-    ret += label_tag name_for_setting_attributes("display"), l(:label_column_in_backlog)
-    ret += select_tag name_for_setting_attributes("display"), options_for_select(position_display_options, setting.display)
-    ret += hidden_field_tag name_for_setting_attributes("id"), setting.id if setting.id
-    ret += hidden_field_tag "project_id", project.id
-    ret += "</p>"
-
-    ret.html_safe
+    content_tag :div, class: 'form--field' do
+      [
+        styled_label_tag(name_for_setting_attributes('display'), l(:label_column_in_backlog)),
+        styled_select_tag(name_for_setting_attributes('display'), options_for_select(position_display_options, setting.display)),
+        version_hidden_id_field(setting),
+        hidden_field_tag('project_id', project.id)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.